ivy_models/unet/unet_parts.py

```diff
@@ -0,0 +1,66 @@
+import ivy
+class DoubleConv(ivy.Module):
```

Comment:
a small request, is it possible to rename the custom internal layers such that it is for unet usage?
eg, UNetDoubleConv, UNetUp, UNetDown, etc.